build.gradle

```diff
@@ -982,7 +987,13 @@ project('standalone') {
             classpath += files(systemProperties.get("extDirs"))
         }
 
-        jvmArgs "-Xdebug", "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=n"
+        def JDWP_DEFAULT_PORT = 8050
+        def jvmDebugPort = System.getenv("JDWP_PORT")
+        if (jvmDebugPort != null && jvmDebugPort != "") {
```

Comment:
it might be a good idea to use regex to validate `jvmDebugPort` contains only numbers.
Something like `String regex = "\\d+";` and then `jvmDebugPort.matches(regex)` to avoid max possible junk value

Reply:
Thanks Shashwat, will it be good if we use regex that validate any given port is numeric and between the range of 0 and 65535(If I am not wrong this should be the port range)? 
If validation fails then default port will be considered ie 8050.

Reply:
Using the default value if the value provided by the user is incorrect is the idea. The numeric range for the ports can be between 0 to 65535. So, to me it makes sense to use the default value if the value provided is not numeric and not in range of valid ports.

Reply:
The changes are incorporated as per suggestion